setup.py

```diff
@@ -2,30 +2,33 @@
 import codecs
 import sys
 
-import setuptools
+from setuptools import find_packages, setup
 
-import sprockets.amqp
+from sprockets.mixins import amqp
 
+version = amqp.__version__
+try:
+    with open('LOCAL-VERSION') as f:
+        local_version = f.readline().strip()
+        version = version + local_version
+except IOError:
+    pass
 
-install_requires = []
-setup_requires = []
-tests_require = ['nose>=1.3,<2', 'tornado>3,<5']
-
-if sys.version_info < (3, 0):
-    tests_require.append('mock')
-
-setuptools.setup(
-    name='sprockets.amqp',
-    version=sprockets.amqp.__version__,
-    description='Making logs nicer since 2015!',
+setup(
+    name='sprockets.mixins.amqp',
+    description='Mixin for publishing events to RabbitMQ',
+    version=version,
+    packages=find_packages(),
+    namespace_packages=['sprockets', 'sprockets.mixins'],
+    test_suite='nose.collector',
     long_description=codecs.open('README.rst', encoding='utf-8').read(),
-    url='https://github.com/sprockets/sprockets.amqp.git',
-    author='Dave Shawley',
-    author_email='daves@aweber.com',
-    license='BSD',
-    extras_require={'tornado': ['tornado>3,<5']},
+    install_requires=open('requires/installation.txt').read(),
+    author='AWeber Communications, Inc.',
+    author_email='api@aweber.com',
+    license=codecs.open('LICENSE', encoding='utf-8').read(),
+    url='https://github.com/sprockets/sprockets.mixins.amqp.git',
     classifiers=[
-        'Development Status :: 4 - Beta',
+        'Development Status :: 1 - Planning',
```

Comment:
Is it really ``Planning`` status? ``Development Status :: 3 - Alpha`` seems more appropriate.